src/runtime/entitlements-manager.js

```diff
@@ -43,10 +44,10 @@ export class EntitlementsManager {
     this.win_ = win;
 
     /** @private @const {!../model/page-config.PageConfig} */
-    this.config_ = config;
+    this.pageConfig_ = config;
```

Comment:
In this case, let's rename `config` argument as well to `pageConfig`.

Reply:
will do